abi/rst2pdf-abi.style

```diff
@@ -0,0 +1,132 @@
+"pageSetup" : {
```

Comment:
I wonder if it is worth moving anything "PDF related" under `abi/tools/pdf`, or similar? In there, it would be useful to have also a script that can be used to invoke rst2pdf

Reply:
Agreed, this would be nicer. I'll do this.

Reply:
done. see the updated commit.